taqueria-plugin-taquito/fund.ts

```diff
@@ -0,0 +1,92 @@
+import {
+	getCurrentEnvironment,
+	getCurrentEnvironmentConfig,
+	sendAsyncErr,
+	sendJsonRes,
+	sendWarn,
+} from '@taqueria/node-sdk';
+import { TezosToolkit } from '@taquito/taquito';
+import {
+	configureToolKitForNetwork,
+	FundOpts as Opts,
+	getDeclaredAccounts,
+	getEnvTypeAndNodeConfig,
+	getNetworkInstantiatedAccounts,
+} from './common';
+import { ContractInfo, performTransferOps } from './transfer';
+
+type TableRow = {
+	accountAlias: string;
+	accountAddress: string;
+	mutezFunded: string;
+};
+
+const getAccountsInfo = (
+	parsedArgs: Opts,
+	tezos: TezosToolkit,
+	instantiatedAccounts: Record<string, any>,
+): Promise<ContractInfo[]> =>
+	Promise.all(
+		Object.entries(instantiatedAccounts)
+			.map(async (instantiatedAccount: [string, any]) => {
+				const alias = instantiatedAccount[0];
+				const aliasInfo = instantiatedAccount[1];
+
+				const declaredMutez: number | undefined = getDeclaredAccounts(parsedArgs)[alias];
+				const currentBalanceInMutez = (await tezos.tz.getBalance(aliasInfo.publicKeyHash)).toNumber();
+				const amountToFillInMutez = declaredMutez ? Math.max(declaredMutez - currentBalanceInMutez, 0) : 0;
+
+				if (!declaredMutez) {
+					sendWarn(
+						`Warning: ${alias} is instantiated in the target environment but not declared in the root level "accounts" field of ./.taq/config.json so ${alias} will not be funded as we don't have a declared tez amount set there for ${alias}\n`,
```

Comment:
Technical writing should be in second person narrative, so avoid terms like "we", "us", etc

In second person, it's mostly about using "you"